resources/js/mixins/url.js

```diff
@@ -0,0 +1,354 @@
+import Constants from '../constants';
+
+export default {
+    methods:{
+        /*
+         * Plentymarkets method for getting url params
```

Comment:
- Be more specific here. The function name already says that it is getting the URL params but the doc block doesn't describe what it actually does.
- `urlParams` is missing in the doc block